package.json

```diff
@@ -100,5 +108,9 @@
         "webpack": "^4.19.1",
         "webpack-cli": "^3.1.2",
         "webpack-merge": "^4.1.5"
+    },
+    "engines": {
```

Comment:
Does this throw warning if you machine doesn't satisfy requirements?

Reply:
for running npm commands, yep. I just put in the latest LTS release.